src/main/augurNodeServer.js

```diff
@@ -25,17 +45,18 @@ function AugurNodeServer(selectedNetwork) {
   this.selectedNetwork = selectedNetwork
   this.augur = new Augur()
   this.appDataPath = appData('augur')
-  this.augurNodeController = new AugurNodeController(this.augur, this.selectedNetwork, this.appDataPath)
+  this.augurNodeController = new AugurNodeController(this.augur, this.selectedNetwork, this.appDataPath, IS_WARP_SYNC)
```

Comment:
Are the constructor args aren't the same as below. Are they both right?

Reply:
missed removing it as a network configuration property, it got promoted to a parameter. fixed it.